velox/functions/prestosql/aggregates/tests/ApproxPercentileTest.cpp

```diff
@@ -24,89 +24,96 @@ namespace facebook::velox::aggregate::test {
 
 namespace {
 
-std::string
-functionCall(bool keyed, bool weighted, double percentile, double accuracy) {
+std::string functionCall(
+    bool keyed,
+    bool weighted,
+    double percentile,
+    double accuracy,
+    int percentileCount) {
   std::ostringstream buf;
   int columnIndex = keyed;
   buf << "approx_percentile(c" << columnIndex++;
   if (weighted) {
     buf << ", c" << columnIndex++;
   }
-  buf << ", " << percentile;
+  buf << ", ";
+  if (percentileCount == -1) {
+    buf << percentile;
+  } else {
+    buf << "ARRAY[";
+    for (int i = 0; i < percentileCount; ++i) {
+      buf << (i == 0 ? "" : ",") << percentile;
+    }
+    buf << ']';
+  }
   if (accuracy > 0) {
     buf << ", " << accuracy;
   }
   buf << ')';
   return buf.str();
 }
 
-std::string label(
-    const char* prefix,
-    bool weighted,
-    bool singleAgg,
-    double percentile,
-    double accuracy) {
-  std::ostringstream buf;
-  buf << prefix;
-  if (weighted) {
-    buf << "_weighted";
-  }
-  if (singleAgg) {
-    buf << "_single_agg";
-  }
-  buf << '_' << percentile << "_pct";
-  if (accuracy > 0) {
-    buf << "_" << accuracy << "_accuracy";
-  }
-  return buf.str();
-}
-
 class ApproxPercentileTest : public AggregationTestBase {
  protected:
   void SetUp() override {
     AggregationTestBase::SetUp();
     random::setSeed(0);
   }
 
+  // TODO: Use `testAggregations` once T123448595 is done.
```

Comment:
Would you file GitHub issue and refer to it here? T123448595 is not accessible outside of Meta.

Reply:
Create #2430 for this